docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -94,7 +95,8 @@ test it on our PC or on another EV3:
       client.subscribe("topic/test")
 
     def on_message(client, userdata, msg):
-      if (msg.payload == "Hello world!"):
+      # Convert payload from binary to ascii-format
+      if (msg.payload.decode('ascii') == "Hello world!"):
```

Comment:
And unless there is a specific reason for using ascii encoding, `msg.payload.decode()` would be simpler.